src/core/apparmor-setup.c

```diff
@@ -20,16 +20,10 @@ int mac_apparmor_setup(void) {
         int r;
 
         if (!mac_apparmor_use()) {
-                log_debug("Skipping AppArmor initialization: not supported by the kernel or disabled.");
+                log_debug("Skipping AppArmor initialization: not supported by the kernel, is disabled or libapparmor is not installed.");
```

Comment:
"not supported by the kernel, disabled, or libapparmor not installed"

Reply:
-> #39837.